src/lib/konnectors.js

```diff
@@ -147,7 +147,7 @@ export function run (cozy, konnector, account, timeout = 120 * 1000) {
 function waitForJobFinished (cozy, job, timeout) {
   return new Promise((resolve, reject) => {
     const idTimeout = setTimeout(() => {
-      reject(new Error('Job timed out'))
+      reject('JOB_TIMEOUT')
```

Comment:
My linter always want me to reject with an Error object. I suggest:
```js
reject(new Error('JOB_TIMEOUT'))
```
I am still thinking that we should all discuss about our errors, what we expect from them and how to formalize them. ping @CPatchane @lespacedunmatin